api/src/pcapi/routes/backoffice/chronicles/blueprint.py

```diff
@@ -148,4 +224,76 @@ def unpublish_chronicle(chronicle_id: int) -> utils.BackofficeResponse:
         chronicle=chronicle,
     )
     flash(f"La chronique {chronicle_id} à été dépubliée", "success")
-    return redirect(url_for("backoffice_web.chronicles.list_chronicles"), code=303)
+    return redirect(request.referrer or url_for("backoffice_web.chronicles.list_chronicles"), code=303)
+
+
+@chronicles_blueprint.route("/<int:chronicle_id>/attach-product", methods=["POST"])
+@atomic()
+@permission_required(perm_models.Permissions.MANAGE_CHRONICLE)
+def attach_product(chronicle_id: int) -> utils.BackofficeResponse:
+    if not FeatureToggle.WIP_ENABLE_CHRONICLES_IN_BO.is_active():
+        raise NotFound()
+
+    form = forms.AttachProductForm()
+    if not form.validate():
+        mark_transaction_as_invalid()
+        flash(utils.build_form_error_msg(form), "warning")
+        redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+    chronicle = chronicles_models.Chronicle.query.get_or_404(chronicle_id)
+    product = offers_models.Product.query.filter(offers_models.Product.extraData["ean"].astext == form.ean.data).first()
+
+    if not product:
+        mark_transaction_as_invalid()
+        flash("Aucune œuvre n'à été trouvée pour cet EAN")
+        return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
+
+    chronicle.products.append(product)
+    db.session.add(product)
+    db.session.flush()
+
+    flash(f"{product.name} a été rattaché à la chronique", "success")
+    return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id), code=303)
```

Comment:
Revenons au même onglet, affichant les œuvres associées permettant de visualiser l'ajout :
```suggestion
    return redirect(url_for("backoffice_web.chronicles.details", chronicle_id=chronicle_id, active_tab="book"), code=303)
```